addons/website_blog/static/src/js/website_blog.js

```diff
@@ -1,84 +1,140 @@
-        } else if ($(this).hasClass('o_linkedin_complete')){
-            url = 'https://www.linkedin.com/shareArticle?mini=true&url='+window.location.href+'&title='+blog_title_complete;
-        } else {
-            url = 'https://plus.google.com/share?url='+window.location.href;
-        }
-    }
-    else {
-        var blog_post = $(this).parents("[name='blog_post']");
-        var blog_post_title = blog_post.find('.o_blog_post_title').html() || '';
-        var blog_article_link = blog_post.find('.o_blog_post_title').parent('a').attr('href');
-        if ($(this).hasClass('o_twitter')) {
-            url = 'https://twitter.com/intent/tweet?tw_p=tweetbutton&text=Amazing blog article : '+blog_post_title+"! "+window.location.host+blog_article_link;
-        } else if ($(this).hasClass('o_facebook')){
-            url = 'https://www.facebook.com/sharer/sharer.php?u='+window.location.host+blog_article_link;
-        } else if ($(this).hasClass('o_linkedin')){
-            url = 'https://www.linkedin.com/shareArticle?mini=true&url='+window.location.host+blog_article_link+'&title='+blog_post_title;
-        } else if ($(this).hasClass('o_google')){
-            url = 'https://plus.google.com/share?url='+window.location.host+blog_article_link;
-        }
-    }
-    window.open(url, "", "menubar=no, width=500, height=400");
-}
+sAnimations.registry.websiteBlog = sAnimations.Class.extend({
+    selector: '.website_blog',
+    read_events: {
+        'click .cover_footer': '_onNextBlogClick',
+        'click a[href^="#blog_content"]': '_onAnimate',
+        'click .o_twitter': '_onShareArticle',
+        'click .o_facebook': '_onShareArticle',
+        'click .o_linkedin': '_onShareArticle',
+        'click .o_google': '_onShareArticle',
+        'click .o_twitter_complete': '_onShareArticle',
+        'click .o_facebook_complete': '_onShareArticle',
+        'click .o_linkedin_complete': '_onShareArticle',
+        'click .o_google_complete': '_onShareArticle',
+        'click .blog_post_year_collapse': '_onToggleChevron',
+        'mouseenter div.o_blog_post_complete a': '_onHoverBlogPost',
+        'mouseleave div.o_blog_post_complete a': '_onUnhoverBlogPost'
+    },
 
-$(document).ready(function() {
-    if ($('.website_blog').length) {
-        $(".js_tweet, .js_comment").share({});
-        $('.cover_footer').on('click', page_transist);
-        $('a[href^="#blog_content"]').on('click', animate);
-        $('.o_twitter, .o_facebook, .o_linkedin, .o_google, .o_twitter_complete, .o_facebook_complete, .o_linkedin_complete, .o_google_complete').on('click', shareArticle);
-        /* Archives years collapse */
-        $('.blog_post_year_collapse').on('click', function() {
-            $(this).find('i.fa').toggleClass('fa-chevron-down fa-chevron-right');
-        });
+    /**
+     * @override
+     * @param {Object} parent
+     */
+    start: function (parent) {
+        if ($('.website_blog').length) {
```

Comment:
That condition is useless. The animation selector is already `.website_blog`, so when you are in the `start` method, it means that condition is always true.